apps/condo/domains/ticket/utils/handlers.js

```diff
@@ -247,6 +251,8 @@ const sendTicketCommentNotifications = async (requestData) => {
             user: { id: clientId },
             property: { id: propertyId },
             organization: { id: organizationId },
+            unitName,
```

Comment:
You need to check that we have a constrain for that

Reply:
Please add the constrain in other PR

Reply:
Added ticket for that DOMA-2916